.github/workflows/release.yml

```diff
@@ -45,6 +45,12 @@ jobs:
           MAVEN_CENTRAL_TOKEN: ${{ secrets.SONATYPE_PASSWORD }}
           MAVEN_GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I just realised this needs to be a Personal Access Token. You can replace it with `secrets.PAT`